libevm/lib/evm.go

```diff
@@ -40,10 +44,31 @@ type EvmContext struct {
 }
 
 type TraceOptions struct {
-	EnableMemory     bool `json:"enableMemory"`
-	DisableStack     bool `json:"disableStack"`
-	DisableStorage   bool `json:"disableStorage"`
-	EnableReturnData bool `json:"enableReturnData"`
+	EnableMemory     bool          `json:"enableMemory"`
+	DisableStack     bool          `json:"disableStack"`
+	DisableStorage   bool          `json:"disableStorage"`
+	EnableReturnData bool          `json:"enableReturnData"`
+	Tracer           *string       `json:"tracer"`
```

Comment:
Prefer using just `string` (not a pointer), and check against empty string instead of `nil`.